src/main/java/com/rarchives/ripme/ripper/rippers/DeviantartRipper.java

```diff
@@ -267,6 +267,8 @@ public String getUsername(Document doc) {
             try {
                 String imageURL = doc.select("span").first().attr("data-super-full-img");
                 if (!imageURL.isEmpty() && imageURL.startsWith("http")) {
+                    String page = doc.select("span").first().attr("href");
+                    imageURL = smallToFull(imageURL,page);
```

Comment:
Ripping the album https://www.deviantart.com/justgenitals/gallery/64358240/SPECIAL-MinamiShots I'm not seeing any difference between the images downloaded using smallToFull and the ones not, can you give me a test link so I can confirm that the ripper now downloads full sized images?